src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -86,6 +87,11 @@ void GraphOptimizer::ApplyCommonGraphOptimizations(Graph &graph) {
     MergeConvertAndScaleShift(graph);
     graph.RemoveDroppedNodes();
 
+    // This optimization forces to nodes to have fp16 on directly inputs and allows them to handle precision conversion based on internal logic.
```

Comment:
1. Lets move comment inside the itself pass.
2. Wording: "This optimization fuses Convert (fp16 -> bf16/fp32) on weights directly to FC input to allow precision conversion handling based on internal logic (e.g. fuse conversion with weights reordering)"